doc/apiref.rst

```diff
@@ -521,6 +525,18 @@ A JSON array is an ordered collection of other JSON values.
    *value*. This is useful when *value* is newly created and not used
    after the call.
 
+.. function:: int json_array_append_unique(json_t *array, json_t *value)
+
+   Appends *value* to the end of *array*, but only if *array* does not contain
+   *value* unique. Returns 0 on success and -1 on error.
```

Comment:
Grammar mistake, should be something like "but only if *array* does not already contain *value*."